test/jet.jl

```diff
@@ -0,0 +1,9 @@
+import JET, JuliaHub
+using Test
+
+jet_mode = Symbol(get(ENV, "JULIAHUB_TEST_JET", "typo"))
+@info "Running JET.jl in mode=:$(jet_mode)"
+
+@testset "JET" begin
+    JET.test_package("JuliaHub"; target_defined_modules=true, mode=jet_mode)
```

Comment:
`JET.test_package("JuliaHub"; target_defined_modules=true, mode=jet_mode, toplevel_logger=nothing)` if you want to disable the `[toplevel-info]` message.

Reply:
Or probably JET should disable it by default for `test_package`.